### ISSUE TITLE
Any performance differences between 3.10 and latest?

### ISSUE THREAD

**Issue by mammuthus** [schweikert/fping#155]
Have Centos 7 with fping 3.10 (use it with Zabbix - a lot of pingers really).
I want to optimize monitoring performance (decrease fping cpu comsumption), so I want to know if it makes sense to install latest fping (4.2) or nothing will change?

**Comment by robotman321** [comment]
I bet you've already decided to go forward or not at this point, but I recently made the same switch 3.15 -> 4.2.. Been dealing with endpoints that respond to ICMP requests no longer getting data back from fping for those endpoints (when 3.15 worked fine before). Currently digging through the changelog to try and understand what might have been changed to cause this odd behavior. (If you've encountered anything i'd low to hear it)

**Comment by robotman321** [comment]
Hah.. so figured it out shortly after posting my prior comment. My use case was something like the following -- `fping -u -C 10 -p 5 -e endpoint` on 3.15 it sent out 10 requests 5ms apart. I changed the value on an endpoint that saw an average of 25ms response and found that at `-p 30` i was getting back all of the results. Tried on a public endpoint halfway around the globe from me and confirmed that i had to set the iteration value up to 150 to see the 140ms pings. I checked the help for the two version and they have the same wording :/ next step is to check to see if this was a feature of the old version that got fixed in the iterations up to 4.15... Other than that it seems to be performing quite well.
